predicate/cli/util.py

```diff
@@ -0,0 +1,57 @@
+"""
+Copyright 2022 Gravitational, Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import ast
+import os
+from jinja2 import FileSystemLoader, Environment, select_autoescape
+
+def get_classname(policy: str):
+    """
+    Get class name from policy file. Expects one class per poliy file.
+    """
+
+    parsed = ast.parse(policy)
```

Comment:
My reason for doing this is to make the classnames not specific to Teleport. This will be more meaningful when we support checking policies outside of Teleport ecosystem in the future. Since this implementation extracts the first available classname in the python file, I think it should be okay as long as user follows the pattern of writing policy within the first declared class. We can explain this in docs. 

But happy to know if you think this will cause a problem later.

Reply:
I have now used `run_path` to grab the classname and verify if the `p` property is of Policy class as `hasattr(module, 'p') and isinstance(module.p, Policy)`. It's much simpler than using ast. Thanks for the tip.